### DISCUSSION TITLE
The popup toolbar is hidden in dialog!

### DISCUSSION THREAD

**Discussion by yetrun** [tinymce/tinymce#8633] [General]
How to make it appear？

**Comment by lostkeys** [comment]
@yetrun we will need a bit more information to understand what you mean, sorry.

**Reply by yetrun** [reply]
Sorry, I make a supply for you. I want to place a richtext editor in a dialog and use the `quickbars_selection_toolbar`. But the toolbar seems to be hidden below the dialog but not shown on it. Is there a method to make the `quickbars_selection_toolbar` to appear on the dialog, such as adjust it `z-index` or other?

Below is the pseudocode for illustrating：

```html
<div class="dialog" style="z-index: 100">
  <div id="editor"></div>
</div>

<script>
TinyMCE.init({
  id: "editor",
  quickbars_selection_toolbar: "bold italic underline | bullist numlist outdent indent | removeformat"
})
</script>
```
